### ISSUE TITLE
Default _extensions property in wrong place in course.json

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#3348]
### Subject of the issue
It should be back one level, against `_accessibility`, not inside it.

https://github.com/adaptlearning/adapt_framework/blob/99db7d7e82ac7fd3734f2651bab89a73de26a4a0/src/course/en/course.json#L67-L84

Original issue: https://github.com/adaptlearning/adapt-contrib-core/pull/239
